two_muscle_case/ellipsoid_muscles/README.md

```diff
@@ -11,3 +11,24 @@ To run a single simulation of contracting the muscles, go to build_release and r
 ```
 ./two_muscles_contraction ../settings_contraction.py
 ```
+
+## Results
+To answer the question how coupled ellipsoid muscles behave we have a few results.
+
+The first plot shows the difference between a single muscle and a coupled muscle: 
+
+![](../../figures/comparison_coupled_ellipsoid_muscle_single_muscle.png)
+
+The blue plot is the length over time of an ellipsoid muscle that contracts while being connected to another ellipsoid muscle via a spring with constant $k=500$. The orange plot is the length over time of a single ellipsoid without connections to other muscles. As we can see, the connection dampens the muscle's bouncing effect after the completed contraction. Also, the connected muscle does not contract as much as the single one does.
+
+If we now look at two connected ellipsoid muscles with a spring constant $k=500$, we get the following plot for the lengths over time of the connected muscles:
+
+![](../../figures/1ms_k_500_ellipsoid.png)
+
+The orange line here is the same as the blue line in the plot before. The blue line here is the coupled muscle that is not being activated and only stretched by muscle 2. The spike-effect in the blue line happens because the time steps are chosen quite large.
```

Comment:
To me spike-effect is not very clear, I think the word oscillations fits better. Do we actually have proof that they go away with a smaller time step?